lib/pages/chat/events/message.dart

```diff
@@ -481,7 +481,7 @@ class Message extends StatelessWidget {
 
   Widget _messageSelectedWidget(BuildContext context, Widget child) {
     return Container(
-      padding: EdgeInsets.symmetric(horizontal: selected ? 8 : 0, vertical: selected ? 4 : 0),
+      padding: EdgeInsets.symmetric(vertical: selected ? 1 : 0, horizontal: selected ? 8 : 0),
```

Comment:
```suggestion
      padding: selected
          ? const EdgeInsets.symmetric(vertical: 1, horizontal: 8)
          : EdgeInsets.zero,
```

Reply:
What is different?

Reply:
just coding style, easier to understand, anyway use `const` is better right?